README.md

```diff
@@ -1,5 +1,5 @@
 # GRChombo
-GRChombo is a new open-source code for numerical general relativity simulations.
+GRChombo is an open-source code for numerical general relativity simulations.
```

Comment:
I think we should remove the word "general" given that some users use GRChombo for modified gravity and some users use no gravity!

```suggestion
GRChombo is an open-source code for numerical relativity simulations.
```